pages/common/touch.md

```diff
@@ -21,4 +21,4 @@
 
 - Create multiple files:
 
-`touch {{path/to/file{1,2,3}.txt}}`
+`touch {{path/to/file1 path/to/file2 ...}}`
```

Comment:
:thinking: This is a simple command, maybe we add a new example with `file{1,2,3}` to explain that bash feature too. wdyt?

Reply:
It's a good idea to note about shell specific features. 👍🏻 But there is a problem: we have to decide what features are noted about in tldr pages to make pages more standardized. There are too many pages without notion of brace expansion. It is assumed (as I understand) that user adapt suggested syntax to a specific shell. I suggest orient to a POSIX shell and suggest just some of it's features explicitly (as a separate examples or in braces in code descriptions).

[Here][bashisms] you can find bash-specific features. Brace expansion is one of them: and I am not sure we have to stick to Bash shell in some pages. There are other shells too with their specific syntax so it's not clear why not to note about them too. But in this case we are going to make our descriptions very long. 😄 

[bashisms]: https://mywiki.wooledge.org/Bashism

Reply:
I like our current standard for "multiple files", its clear for users to understand and the people can often almost copy the listed commands.

Reply:
Well, what about using brace expansion everywhere for arbitrary number of arguments? For package names, user names, file names and etc. 🤔